OSDOCS-16077 [NETOBSERV] 1.9.3 release notes (#98549)
openshift/openshift-docs: observability/network_observability/network-observability-release-notes-1-9-3.adoc

//Network Observability Operator Release Notes
:_mod-docs-content-type: ASSEMBLY
[id="network-observability-release-notes-1-9-3"]
= Network Observability Operator release notes 1.9.3

:context: network-observability-operator-release-notes-v1-9-3
include::_attributes/common-attributes.adoc[]

toc::[]

> The Network Observability Operator enables administrators to observe and analyze network traffic flows for {product-title} clusters.

Reviewer: ```suggestion
With the Network Observability Operator, administrators can observe and analyze network traffic flows for {product-title} clusters.
```
See IBM Style Word Usage advice about "enable": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#enable
